crates/trigger/src/lib.rs

```diff
@@ -95,6 +95,27 @@ impl<Executor: TriggerExecutor> TriggerExecutorBuilder<Executor> {
                 builder.add_host_component(outbound_redis::OutboundRedisComponent)?;
                 builder.add_host_component(outbound_pg::OutboundPg::default())?;
                 builder.add_host_component(outbound_mysql::OutboundMysql::default())?;
+
+                let parent_dir = match dirs::home_dir() {
+                    Some(home) => home.join(SPIN_HOME),
+                    None => PathBuf::new(),
+                };
+
+                // TODO: Use relevant portion of app config here.
+
+                builder.add_host_component(key_value::KeyValueComponent::new(
+                    key_value::Config {
+                        configs: [(
+                            "".to_owned(),
+                            key_value::ImplConfig::Sqlite(key_value::sqlite::Config::Path(
+                                parent_dir.join("sqlite_key_value.db"),
```

Comment:
Maybe add a comment here that for now only the file based SQLite implementation can be used. There is no current way to use in memory right, since we are only using default? This could also be a note around the `sqlite::Config` object